docs/Gemfile.lock

```diff
@@ -42,22 +42,22 @@ GEM
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
     liquid (4.0.4)
-    listen (3.8.0)
+    listen (3.9.0)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
     mercenary (0.4.0)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
-    public_suffix (5.0.4)
-    rake (13.1.0)
+    public_suffix (5.0.5)
+    rake (13.2.1)
     rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
     rexml (3.2.6)
     rouge (3.30.0)
     safe_yaml (1.0.5)
-    sass-embedded (1.69.7)
-      google-protobuf (~> 3.25)
+    sass-embedded (1.63.6)
```

Comment:
hmm why is this downgraded?

Reply:
Probably because I was running a different version of Ruby and/or Bundler when I last updated the lock file.

But as part of this PR I made sure to run Ruby 2.7 and Bundler 2.4.22 to match the CI environment.

Keep in mind that Bundler selects the versions in this lock file based on the Ruby version and required dependencies.